# PosicionApoyo.srv
---
geometry_msgs/Pose pose